benchmark/buffers/buffer-read.js

```diff
@@ -33,9 +35,13 @@ function main(conf) {
   var fn = 'read' + conf.type;
 
   buff.writeDoubleLE(0, 0, noAssert);
+
   var testFunction = new Function('buff', [
     'for (var i = 0; i !== ' + len + '; i++) {',
-    '  buff.' + fn + '(0, ' + JSON.stringify(noAssert) + ');',
+    '  if (\'' + fn + '\' === \'readIntLE\' ' +
```

Comment:
The if branch should be moved outside `new Function` to avoid the runtime cost. Also would be easier to read if we use template literals here, something like:

```js
var call;
if (fn === 'readIntLE' || conf.type === 'readIntBE') {
  call = `buff.${fn}(0, 1, ${noAssert})`;
} else {
  call = `buff.${fn}(0, ${noAssert})`;
}
var testFunction = new Function('buff',
  ` for (var i = 0; i !== ${len}; ++i) {
    ${call};
  }`);
```